troposphere/static/js/components/modals/instance/launch/steps/BasicLaunchStep.react.js

```diff
@@ -15,43 +15,15 @@ export default React.createClass({
                     <div className="col-md-6">
                         <h3 className="t-title">Basic Info</h3>
                         <hr/>
-                        <BasicInfoForm
-                            image={this.props.image}
-                            project={this.props.project}
-                            projectList={this.props.projectList}
-                            imageVersionList={this.props.imageVersionList}
-                            imageVersion={this.props.imageVersion}
-                            onNameChange={this.props.onNameChange}
-                            onVersionChange={this.props.onVersionChange}
-                            onProjectChange={this.props.onProjectChange}
-                        />
+                        { React.createElement(BasicInfoForm, this.props) }
```

Comment:
Just to capture something we talked about in _code review_, I believe we could do: 
```
  <BasicInfoForm {...this.props} />
```

source: https://facebook.github.io/react/docs/jsx-spread.html#spread-attributes